src/test/librbd/fsx.cc

```diff
@@ -1405,12 +1405,12 @@ dowrite(unsigned offset, unsigned size)
 		       (debug &&
 		       (monitorstart == -1 ||
 			(offset + size > monitorstart &&
-			(monitorend == -1 || offset <= monitorend))))))
+			 (monitorend == -1 || (long)offset <= monitorend))))))
```

Comment:
yeah, it already has this C style casting. and i found that `fsx.cc` originally comes from AFS and was written in C. but we adapted to our usage and changed it to fsx.**cc**.